spec/factories/application_choice.rb

```diff
@@ -40,7 +46,7 @@
 
       after(:build) do |application_choice, _evaluator|
         application_choice.status = :awaiting_provider_decision
-        application_choice.interviews << build(:interview, :cancelled, provider: application_choice.provider)
+        application_choice.interviews << build(:interview, :cancelled, provider: application_choice.current_course_option.course.provider)
```

Comment:
Is there we can't use `current_provider` and similar in the factory file?

Reply:
Should be fine